radicle-cob/src/test/crypto.rs

```diff
@@ -0,0 +1,229 @@
+use std::{
```

Comment:
We can depend on radicle-crypto now!

Reply:
Oh ya, we do! I just forgot to delete this file :smile: